### DISCUSSION TITLE
installation openshift-install stuck/hang on AWS #1

### DISCUSSION THREAD

**Discussion by andripras** [okd-project/okd#1810] [General]
i'm going to install Openshift OCP on EC2 Instance using Bastion, when I start the installation with command "openshift-install create cluster --dir <directory> --log-level=debug", just stuck like this and there is no continuation:

DEBUG OpenShift Installer 4.12.40
DEBUG Built from commit 576d8157addb36d6f8fc27e0db4d8559d6889ac4
DEBUG Fetching Metadata...
DEBUG Loading Metadata...
DEBUG   Loading Cluster ID...
DEBUG     Loading Install Config...
DEBUG       Loading SSH Key...
DEBUG       Loading Base Domain...
DEBUG         Loading Platform...
DEBUG       Loading Cluster Name...
DEBUG         Loading Base Domain...
DEBUG         Loading Platform...
DEBUG       Loading Networking...
DEBUG         Loading Platform...
DEBUG       Loading Pull Secret...
DEBUG       Loading Platform...
INFO Credentials loaded from the "okd-admin" profile in file "/root/.aws/credentials"

i choose the install-config.yaml from this link https://docs.okd.io/4.13/installing/installing_aws/installing-aws-private.html#installation-aws-config-yaml_installing-aws-private.
Is there anything missing before starting the installation on EC2? Please Help Me.

Thank You,

**Comment by vrutkovs** [comment]
Please attach [installation log](https://docs.okd.io/latest/installing/installing-troubleshooting.html#installing-getting-debug-information_installing-troubleshooting)

**Reply by andripras** [reply]
Hi @vrutkovs thanks for replying.

instalation logs like this only:

time="2023-11-23T00:26:01Z" level=debug msg="OpenShift Installer 4.12.0-0.okd-2023-04-16-041331"
time="2023-11-23T00:26:01Z" level=debug msg="Built from commit 4e5513ad5a5887a8036ba74ca48b1fa4eafe76cc"
time="2023-11-23T00:26:01Z" level=debug msg="Fetching Metadata..."
time="2023-11-23T00:26:01Z" level=debug msg="Loading Metadata..."
time="2023-11-23T00:26:01Z" level=debug msg="  Loading Cluster ID..."
time="2023-11-23T00:26:01Z" level=debug msg="    Loading Install Config..."
time="2023-11-23T00:26:01Z" level=debug msg="      Loading SSH Key..."
time="2023-11-23T00:26:01Z" level=debug msg="      Using SSH Key loaded from state file"
time="2023-11-23T00:26:01Z" level=debug msg="      Loading Base Domain..."
time="2023-11-23T00:26:01Z" level=debug msg="        Loading Platform..."
time="2023-11-23T00:26:01Z" level=debug msg="        Using Platform loaded from state file"
time="2023-11-23T00:26:01Z" level=debug msg="      Using Base Domain loaded from state file"
time="2023-11-23T00:26:01Z" level=debug msg="      Loading Cluster Name..."
time="2023-11-23T00:26:01Z" level=debug msg="        Loading Base Domain..."
time="2023-11-23T00:26:01Z" level=debug msg="        Loading Platform..."
time="2023-11-23T00:26:01Z" level=debug msg="      Using Cluster Name loaded from state file"
time="2023-11-23T00:26:01Z" level=debug msg="      Loading Networking..."
time="2023-11-23T00:26:01Z" level=debug msg="        Loading Platform..."
time="2023-11-23T00:26:01Z" level=debug msg="      Using Networking loaded from state file"
time="2023-11-23T00:26:01Z" level=debug msg="      Loading Pull Secret..."
time="2023-11-23T00:26:01Z" level=debug msg="      Using Pull Secret loaded from state file"
time="2023-11-23T00:26:01Z" level=debug msg="      Loading Platform..."
time="2023-11-23T00:26:01Z" level=info msg="Credentials loaded from the \"default\" profile in file \"/root/.aws/credentials\""

**Comment by vrutkovs** [comment]
Also
>DEBUG OpenShift Installer 4.12.40

that doesn't look like OKD installer. Please check that you used the installer from [Releases](https://github.com/okd-project/okd/releases) page or extracted installer from the OKD payload

**Reply by andripras** [reply]
im sorry before. this last installation stuck like this:

DEBUG OpenShift Installer 4.12.0-0.okd-2023-04-16-041331
DEBUG Built from commit 4e5513ad5a5887a8036ba74ca48b1fa4eafe76cc
DEBUG Fetching Metadata...
DEBUG Loading Metadata...
DEBUG   Loading Cluster ID...
DEBUG     Loading Install Config...
DEBUG       Loading SSH Key...
DEBUG       Using SSH Key loaded from state file
DEBUG       Loading Base Domain...
DEBUG         Loading Platform...
DEBUG         Using Platform loaded from state file
DEBUG       Using Base Domain loaded from state file
DEBUG       Loading Cluster Name...
DEBUG         Loading Base Domain...
DEBUG         Loading Platform...
DEBUG       Using Cluster Name loaded from state file
DEBUG       Loading Networking...
DEBUG         Loading Platform...
DEBUG       Using Networking loaded from state file
DEBUG       Loading Pull Secret...
DEBUG       Using Pull Secret loaded from state file
DEBUG       Loading Platform...
INFO Credentials loaded from the "default" profile in file "/root/.aws/credentials"

whats wrong with my bastion? is there anything missing from the prereqition in aws? I already allow ec2 permission, inject iam role account in bastion, and private domain already set on route53. please help me @vrutkovs .

thank you..
